dev-docs/bidders/taboola.md

```diff
@@ -3,6 +3,7 @@ layout: bidder
 title: Taboola
 description: Prebid Taboola Bidder Adapter
 pbjs: true
+pbs: true
```

Comment:
Please also add the following missing header values:
```
pbs_app_supported: true/false
fpd_supported: true/false
ortb_blocking_supported: true/partial/false
multiformat_supported: will-bid-on-one/will-bid-on-any/will-not-bid
deals_supported: true/false
userId: (list of supported vendors)
```

Reply:
@bretg 
added relevant entities to header.

Reply:
thanks - is there a reason you didn't want to add pbs_app_supported or userId?

Reply:
thanks. the default for most values is "check with bidder".